### PR TITLE
Menu entries for improved accessibility.

### PR DESCRIPTION
**Pull request by matkoch**
Added menu entries:
- (F)ile History
- (R)eset File Changes

In order to have more useful mnemonics, I ...
- Removed shortcut for "Format Patch" completely
- Changed "Clone repository" to have the same as "Create repository" (being "y")

### CODE REVIEW

File: GitExtensionsVSIX/GitExtensionsPackage.vsct [hunk 1]
```diff
@@ -104,7 +104,7 @@
         <Icon guid="guidImages" id="clone"/>
         <Strings>
           <CommandName>GitExt.Clone</CommandName>
-          <ButtonText>Clone &amp;repository</ButtonText>
+          <ButtonText>Clone repositor&amp;y</ButtonText>
         </Strings>
```
Comment (jbialobr) [inline GitExtensionsVSIX/GitExtensionsPackage.vsct]: There is already binding for `y` in `Create new repositor&amp;y` is it intended?

Comment (matkoch) [inline GitExtensionsVSIX/GitExtensionsPackage.vsct]: Yes it was intended.

Comment (jbialobr) [inline GitExtensionsVSIX/GitExtensionsPackage.vsct]: Sorry, I just noticed you wrote that in PR description

**Comment by jbialobr** [comment]
Do you know if there is a way to add a shortcut for the GitExt menu in the main menu?

**Comment by matkoch** [comment]
What do you mean? In the main menu? There already is an entry? :smiley:

**Comment by jbialobr** [comment]
When you press the alt key, then there apears an underscore under a letter for each of the menu items, but not for GitExt. See the screenshot below.
2016-08-04 17_04_11-gitextensions vs2015 running - microsoft visual studio

**Comment by matkoch** [comment]
Strange, I have the `I` underlined.

capture

**Comment by jbialobr** [comment]
The shortcut does not show when I added `&` to the `ButtonText`. It appeared when I also added `&` to the `CommandName`. https://github.com/gitextensions/gitextensions/commit/5743ea6949c96a10971ba759a1217efa96c741df

**Pull request merged**